pkg/Cpanel/Security/Advisor/Assessors/Apache.pm

```diff
@@ -410,4 +410,18 @@ sub _grsecurity_symlink_protection {
     return 1;
 }
 
+my $httpd;
+
+sub _get_httpd_path {
+    return $httpd if defined $httpd;
+    $httpd = '/usr/local/apache/bin/httpd';
+    if ( defined &Cpanel::Config::Httpd::is_ea4 ) {
```

Comment:
hmm, must have worked by virtue of being loaded already (like several modules in this package). will add it, thanks